package.json

```diff
@@ -38,7 +38,7 @@
     "@hot-loader/react-dom": "17.0.1",
     "@react-pdf/renderer": "2.0.0",
     "@reduxjs/toolkit": "1.5.1",
-    "@tailwindcss/postcss7-compat": "^2.1.2",
+    "@tailwindcss/postcss7-compat": "2.1.2",
```

Comment:
It could be resolved when we merge `new_development` into master.
As long as you changed it in this PR, all the tailwind related deps should be updated to avoid confusion when we merge new design into master, you can check it in `new_development`.
Please make sure that you remove this dep, it's not needed anymore because we use `postcss8`

Reply:
Yes, but the `new_development ` will not merge to the master soon, that's why I have to update this to make sure it's consistent in the master branch.

For now, we haven't used any `tailwind`, I think it will not cause any problem when we merge with the `new_developmnet` branch.

And before we merge the `new_development` branch into the `master` branch. I think we should merge the `master` into `new_development` first and when we do this step. I think we can keep all dependencies in the `new_development` and update/remove un-need dependencies of the `master` branch. By this way, it will avoid confusion and it safe to merge.

Reply:
Yes, remove or upgrade same as `new_development` - take the easier way for you, I agree 💯